experimental/dds/tree2/src/feature-libraries/defaultChangeFamily.ts

```diff
@@ -100,13 +100,19 @@ export interface IDefaultEditBuilder {
 	 */
 	sequenceField(field: FieldUpPath): SequenceFieldEditBuilder;
 
-	// TODO: document
+	/**
+	 * Moves a subsequence from one sequence field to another sequence.
+	 *
+	 * Note that in the case where the source and destination are the same, the `destinationIndex` is the final index the first node moved would end up at.
```

Comment:
This makes it sound as though this "use the final index of the first node" rule only applies when the source and destination fields are the same. It applies all the time but happens to be more relevant in the case highlighted below.
```suggestion
	 * Note that the `destinationIndex` is the final index the first node moved should end up at.
```
I think the paragraph that follows ("thus...") takes care of the rest.